### ISSUE TITLE
Request: support hyperlink from view visual object 

### ISSUE THREAD

**Issue by rchevallier** [archi-contribs/jarchi-single-page-html-export#14]
To be able to help the user navigate between views of interest from another view, support the "view visual object" as a hyperlink to the corresponding view (same behavior as in Archi when double clicking it)

I suppose it would be dependent on issue [Allow deeplinks to easily bookmark one specific view](https://github.com/archi-contribs/jarchi-single-page-html-export/issues/7)

**Comment by rvelter** [comment]
I agree with this idea. I have many views that are "navigation views (with togaf structure)" and no links from view visual object in this report like in the classical report. C'est dommage :)
I don't want a direct URL to bookmark (like the deeplink issue) but to be able to navigate between views by clicking on View visual object.
I like the idea to be sharepoint preview compliant.

**Comment by jbsarrodie** [comment]
Hi. TBH I first thought it wouldn't be possible because for this to work javascript has to be used, but some time ago I discovered that javascript can be used and this still works in sharepoint/onedrive (which was the primary reason of creating this single page export script.

I haven't fount time to work on it, but I might find some in the future, so this could be implement for Christmas maybe ;-)

**Comment by rvelter** [comment]
Perfect. it would be a nice Christmas gift :)

**Comment by evlibra** [comment]
Any progress on it? New Year gift? :)

**Comment by evlibra** [comment]
How the description of each element can be seen? Looks like current implementation shows only description of views.
[update]: found it on Elements tab - it shows content of all elements of selected view, with description

**Comment by jbsarrodie** [comment]
> Any progress on it? New Year gift? :)

Sorry but I had other things to sort out and haven't come back to code yet. I'll work on it in the near future as I need some enhancements (deep links), but I don't know when exactly.

**Comment by lavenderb** [comment]
Looking forward to having intra-view links restored :-)

**Comment by rvelter** [comment]
> Sorry but I had other things to sort out and haven't come back to code yet. I'll work on it in the near future as I need some enhancements (deep links), but I don't know when exactly.

Any news ?

**Comment by jbsarrodie** [comment]
> Any news ?

I haven't worked on it because I worked on the default HTML report in Archi itself ("flat" design, deeplink, image zoom, search in the model tree). I'll try to backport some of these features into this jArchi based version later this year.

**Comment by romualdrichard** [comment]
I suppose it's not implemented. This should be great to have it.

**Comment by jbsarrodie** [comment]
> I suppose it's not implemented. This should be great to have it.

Unfortunately, you're supposing right. I'd love to work on it (as I need it myself) but I don't have enough time at the moment (but any contribution welcomed)

**Comment by jaimecerda** [comment]
I'd also love to have this feature.  Most of the projects I've worked with only provide a sharepoint site which this type of report would be a perfect fit.

**Comment by Phillipus** [comment]
Well, the whole point of this being Open Source and community driven is that someone can contribute this feature. So please go ahead.
